pkg/sloop/store/typed/eventcounttable_test.go

```diff
@@ -138,3 +208,26 @@ func (_ *EventCountKey) GetTestKey() string {
 func (_ *EventCountKey) GetTestValue() *ResourceEventCounts {
 	return &ResourceEventCounts{}
 }
+
+func (_ *EventCountKey) SetTestKeys() []string {
+	untyped.TestHookSetPartitionDuration(time.Hour)
+	var keys []string
+
+	//add another key in same partition for testing purpose
+	keys = append(keys, NewEventCountKey(someTs, someKind, someNamespace, someName, someUid+"0").String())
+	gap := 0
+	for i := 'a'; i < 'd'; i++ {
+		// add keys in ascending order
+		keys = append(keys, NewEventCountKey(someTs.Add(time.Hour*time.Duration(gap)), someKind, someNamespace, someName+string(i), someUid).String())
+		gap++
+	}
+	return keys
+}
+
+func (_ *EventCountKey) SetTestValue() *ResourceEventCounts {
+	untyped.TestHookSetPartitionDuration(time.Hour)
+	expectedResult := &ResourceEventCounts{MapMinToEvents: make(map[int64]*EventCounts)}
+	expectedResult.MapMinToEvents[someMinute] = &EventCounts{MapReasonToCount: make(map[string]int32)}
+	expectedResult.MapMinToEvents[someMinute].MapReasonToCount[someReason] = someCount
```

Comment:
This piece is originally from function ` helper_update_eventcount_table`, I decoupled this function to 2 parts, setKey and setValue, since the original unit test all works fine, so I think we can guarantee it should not break things.

The other thing is if you check the unit test here, it all cares about the keys, for the scope of this PR, I think we should be good.

Reply:
If, as you said, the tests are all about keys, then lets eliminate the potential confusion